tensorflow_addons/utils/resource_loader.py

```diff
@@ -37,3 +39,16 @@ def get_path_to_datafile(path):
     """
     root_dir = get_project_root()
     return os.path.join(root_dir, path.replace("/", os.sep))
+
+
+class LazySO:
+    def __init__(self, relative_path):
+        self.relative_path = relative_path
+        self._ops = None
+
+    @property
+    def ops(self):
```

Comment:
Override __getattribute__ hook may a better solution.

Reply:
@fsx950223 thanks for the review! Can you expand on why it may be a better solution ?

Reply:
No need to modify the original code this way。
For example,
```
    return _activation_ops_so.addons_gelu(x, approximate)
```

Reply:
I see what you mean, it's clever :)

Though i'm more in favor of having explicit code rather than having smart code. It's more maintainable and readable.